backend/services/office_hours/ticket.py

```diff
@@ -1,689 +1,280 @@
 
-        # Build List of Creator IDs - Subject and Creators List (For Group)
-        ticket_creator_ids = [subject.id] + [
-            creator.id for creator in oh_ticket.creators if creator.id != subject.id
-        ]
+        # Add new object to table and commit changes
+        self._session.add(oh_ticket_entity)
 
-        creator_section_member_entities = (
-            self._session.query(SectionMemberEntity)
-            .filter(
-                OfficeHoursSectionEntity.id == oh_section.id,
-            )
-            .filter(SectionEntity.office_hours_id == OfficeHoursSectionEntity.id)
-            .filter(
-                SectionMemberEntity.section_id == SectionEntity.id,
-                SectionMemberEntity.member_role == RosterRole.STUDENT,
-                SectionMemberEntity.user_id.in_(ticket_creator_ids),
-            )
-            .distinct()
-            .all()
-        )
+        # Commit so can get ticket id
+        self._session.commit()
 
-        # Raise PermissionError if not all creators are in section
-        if len(creator_section_member_entities) != len(ticket_creator_ids):
-            section_member_user_id = [
-                creator.user_id for creator in creator_section_member_entities
-            ]
-            non_existing_ids = [
-                id for id in ticket_creator_ids if id not in section_member_user_id
-            ]
-            raise PermissionError(
-                f"Creators with ids={non_existing_ids} Are Not Members In Current OH Section id={oh_section.id}"
+        # Now, Associate ticket with Creators
+        for section_member_entity in user_members:
+            self._session.execute(
+                user_created_tickets_table.insert().values(
+                    {
+                        "ticket_id": oh_ticket_entity.id,
+                        "member_id": section_member_entity.id,
+                    }
+                )
             )
 
-        return creator_section_member_entities
-
-    def _check_ticket_creation_time_permissions(
-        self,
-        subject: User,
-        oh_event: OfficeHoursEvent,
-        creators: list[SectionMemberEntity],
-    ) -> None:
-        """Checks if a given user currently has a queued ticket, or has had a called ticket in the past hour.
-        This check will be used to determine whether a student can create a new ticket.
-
-        Args:
-            oh_event: The Office Hours Event of interest
-
-        Raises:
-            PermissionError if student cannot create a ticket right now.
-        """
-
-        # If the student currently has a queued ticket, don't allow new ticket creation
-        if (
-            self._oh_event_svc.check_student_in_queue_status(
-                subject, oh_event
-            ).ticket_id
-            != None
-        ):
-            raise PermissionError(
-                "Cannot create another ticket while currently in the queue."
-            )
```

Comment:
Is this code moved somewhere else or where is the logic for rate limiting ticket creation on a student's part?

Reply:
@KrisJordan nope, re-added this check - thanks for catching this!